Tests/NIOTests/ByteBufferTest.swift

```diff
@@ -1073,22 +1073,37 @@ class ByteBufferTest: XCTestCase {
     }
 
     func testAllocationOfReallyBigByteBuffer() throws {
+        #if !arch(arm) // !32-bit, Raspi/AppleWatch/etc
```

Comment:
I assume this conditional is intended to be temporary? If it's not, can we remove the tests for `arch(arm)` elsewhere in this test function, as they're unreachable.

Reply:
Yes. This fails in a specific ARM memcpy, I hope that we get a fix for that eventually. I raised this in the ARM-community slack. The checks within make the test working until that memcpy error. (maybe that memcpy has additional size or alignment restrictions, no idea yet).